po/es.po

```diff
@@ -2647,55 +2642,56 @@ msgid ""
 "call functions such as `slice::get_unchecked` which does not do bounds "
 "checking."
 msgstr ""
-"El chequeo de límites no puede ser desactivado con el flag de compilador. "
-"Tampoco puede deshabilitarse directamente usando `unsafe`. Sin embargo, "
-"`unsafe` permite que llames a la función como `slice::get_unchecked` que no "
-"brinda chequeos de límites."
+"La comprobación de límites no se puede inhabilitar con una bandera de "
```

Comment:
At least to me 'inhabilitar' has a different meaning (https://www.fundeu.es/consulta/inhabilitar-y-deshabilitar-609/). 'deshabilitar' has widespread use, or 'desactivar' might be also a good replacement